pkgs/development/compilers/julia/0.3.nix

```diff
@@ -33,7 +33,14 @@ stdenv.mkDerivation rec {
         name = "dsfmt-${dsfmt_ver}.tar.gz";
         md5 = "cb61be3be7254eae39684612c524740d";
       };
-    in [ dsfmt_src ];
+
+      llvm_ver = "3.3";
+
+      llvm_src = fetchurl {
```

Comment:
@abbradar Is there a reason we can't continue to pass in `llvm = llvm_33` here and use `llvm_src = llvm.src` instead? It's somewhat cleaner not to have these sources defined many different times.